platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -216,27 +191,28 @@ private void writeOutUsersDotAttributesFile(File userAttributesFile) {
     Map<String, Object> json = null;
     try (InputStream stream = Files.newInputStream(Paths.get(userAttributesFile.toURI()))) {
       json = MAPPER.parser().parseMap(stream);
+    } catch (IOException e) {
+      LOGGER.warn("Unable to read system user attribute file for hostname update.", e);
+    }
+    if (json == null) {
+      LOGGER.debug("Json not found");
```

Comment:
:question: Should we mention the users.attributes file in the log statement?

Reply:
I think that is OK. I wouldn't want to see usernames or passwords written to the log, but the file is public: https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/users.attributes